test/test_torch.py

```diff
@@ -2115,12 +2115,12 @@ def test_inference(default_dtype):
 
     def test_qengnie(self):
         qengines = torch._C._supported_qengines()
-        # [TODO] Enable after the interface change
-        # original_qe = torch._C._get_qengine()
-        # for qe in qengines:
-        #     torch._C._set_qengine(qe)
-        #     assert torch._C._get_qengine() == qe, 'qengine not set successfully'
-        # torch._C._set_qengine(original_qe)
+        original_qe = torch.backends.quantized.engine
+        all_engines = {0 : 'none', 1 : 'fbgemm', 2 : 'qnnpack'}
```

Comment:
should we make this as a global helper function so that users can use it too? I think we can have a `get_qengine_str` in `torch/backends/quantized/__init__.py ` and call this function here.

Reply:
Sounds good.